cmd/findAndReplace/findAndReplace.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"strings"
+)
+
+//TODO
+// findAndReplace to handle both int and string? Currently just strings
```

Comment:
There might be a relatively quick way to implement this on int? I'm trying to think. I thought if you just let the program interpret ints as strings it would still work? If you're looking for parts of the int, e.g. replacing the "5" in "357" to become "397", then you can further slice up the words []string so you get words [][]string, and the second layer would be individual ints interpreted as strings, and you can use a variation of the current for loop on this second layer to do the same "find and replace" function within each string, if that's the functionality you're hoping to achieve?

Reply:
Good point Raven! I tested it and it does handle "int" just fine. At this point I don't have a reason to change part of a column like your example, but that could be written into a future version.